functions/puppetdb_metrics_params.pp

```diff
@@ -105,16 +92,6 @@
       'url'  => 'puppetlabs.puppetdb.storage:name=store-report-time' },
   ]
 
-  #TODO: Track these on a less frequent cadence because they are slow to run
-  $storage_metrics_db_queries = [
```

Comment:
Shouldn't these stay here?  I see you put them in `common.yaml` and I'm not sure why?

Reply:
I movde them because there is no params file any more... its equivalent is `common.yaml`. That said, I can't find where they are used at all so they may just need to go away.

Reply:
should they maybe be in the list of things merged together for of the versions of PE?

Reply:
Yeah, they are part of the list of metrics that gets merged together, so I think they should all be in the same place.  I'm on the fence about where to put them though.  It would make sense to me to put all of them in hiera and have the function only perform the logic around putting them together.  However, `version_specific_metrics` would need to stay in the function since those require some logic to determine.

I guess if one set needs to stay in the function then they all do, unless you see a way for all of them to live in hiera?

Reply:
What I am saying is that I do not see this variable/key referenced anywhere... it seems to be dead code.

Reply:
Ah ok, looking again and it seems you're right.  I think we just remove it entirely then

Reply:
I pushed a commit to remove it.